src/index.ts

```diff
@@ -16,6 +16,6 @@ export default class DataSource {
 
     const url = `${this.sorcererUrlBase}/${this.key}?${paramStr}`;
 
-    return CD.get(url, {}, raw => cb(raw));
+    return CD.get(url, {}, cb);
```

Comment:
Bit unfamiliar with cropduster and `get`, but, if the goal is to not have to lean on callbacks, could you just remove the reference to `cb` here and the arg being passed in?

Reply:
We can, but I was just thinking of backwards compat as some projects/apps are currently using this and they will be expecting `cb`. 

If we remove it and the someone updates the version of this they will need to change their code to remove the callback, which I'm not opposed to but the `getCors` method in the current `CD` library supports callbacks still but just throws a warning so was thinking we should just mimic that:

https://github.com/movableink/cropduster/blob/master/src/cropduster.js#L183